<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BMplayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/image/BMPlayer 512x512.png</normaloff>:/image/image/BMPlayer 512x512.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="3">
     <widget class="QSlider" name="timeLine">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>120</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QSlider" name="volumeLine">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="volumelabel">
      <property name="text">
       <string>Volume：50%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="timelabel">
      <property name="text">
       <string>0.00/0.00</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QWidget" name="MainWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="BMPlayerWidget" name="widget" native="true">
         <property name="whatsThis">
          <string>BMPlayer Widget</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="image.qrc">:/image/image/BMPlayer 200×145.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QWidget" name="contralWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QGridLayout" name="controlLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QToolButton" name="toolButton_play">
            <property name="text">
             <string>play</string>
            </property>
            <property name="icon">
             <iconset resource="image.qrc">
              <normaloff>:/image/image/play.png</normaloff>:/image/image/play.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Space</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QToolButton" name="toolButton_ExitFullscreen">
            <property name="text">
             <string>fullscreen</string>
            </property>
            <property name="icon">
             <iconset resource="image.qrc">
              <normaloff>:/image/image/fullscreen.png</normaloff>:/image/image/fullscreen.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Esc</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="timeLine_Fullscreen">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSlider" name="volumeLine_Fullscreen">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="SoundIcon_Fullscreen">
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  &lt;img src=&quot;:/image/image/volume.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="textlabel">
      <property name="text">
       <string>http://bmplayer.bmysoft.tk</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Url"/>
    <addaction name="actionInternet_Radio"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action"/>
    <addaction name="actionAbout_BMY_Soft"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="action_Contri"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>MainToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="action"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="controlBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>controlBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionPlay_Pause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionFullscreen"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/open.png</normaloff>:/image/image/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/about.png</normaloff>:/image/image/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>About...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/exit.png</normaloff>:/image/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/play.png</normaloff>:/image/image/play.png</iconset>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/fullscreen.png</normaloff>:/image/image/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>fullscreen</string>
   </property>
   <property name="toolTip">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionOpen_Url">
   <property name="text">
    <string>Open &amp;Url</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/image/stop.png</normaloff>:/image/image/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionAbout_BMY_Soft">
   <property name="text">
    <string>About &amp;BMY-Soft...</string>
   </property>
  </action>
  <action name="action_Contri">
   <property name="text">
    <string>&amp;Contributions...</string>
   </property>
  </action>
  <action name="actionInternet_Radio">
   <property name="text">
    <string>Internet &amp;Radio</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BMPlayerWidget</class>
   <extends>QWidget</extends>
   <header>bmplayerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlay_Pause</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFullscreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_fullscreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Url</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_openurl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_BMY_Soft</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>AboutBMYSoft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeLine</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>timeLineChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeLine</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>volumeLineChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_play</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeLine_Fullscreen</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>timeLineChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeLine_Fullscreen</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>volumeLineChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_ExitFullscreen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_fullscreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Contri</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Contribution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInternet_Radio</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>BMplayer_Internet_Radio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BMplayer_open()</slot>
  <slot>BMplayer_start()</slot>
  <slot>BMplayer_pause()</slot>
  <slot>BMplayer_about()</slot>
  <slot>timeLineChanged(int)</slot>
  <slot>BMplayer_fullscreen()</slot>
  <slot>volumeLineChanged(int)</slot>
  <slot>BMplayer_openurl()</slot>
  <slot>BMplayer_stop()</slot>
  <slot>aboutQt()</slot>
  <slot>AboutBMYSoft()</slot>
  <slot>Contribution()</slot>
  <slot>BMplayer_Internet_Radio()</slot>
 </slots>
</ui>
